GUI/main_window.py

```diff
@@ -42,11 +42,11 @@ def __init__(self, *args, **kwargs):
         self.Stack = QStackedWidget(self)
 
         body = Body(width, height)
-        visualizer = Visualizer()
+        #visualizer = Visualizer()
```

Comment:
You can remove lines of code that are commented out

Reply:
I don't think this should be commented out, it's the pod visualizer page

Reply:
I agree with Ryan, the visualizer line shouldn't be commented out or removed

Reply:
Ryan and Kelvin are correct, my mistake. You should uncomment it when committing